hirola/front/views.py

```diff
@@ -446,3 +447,10 @@ def resend_activation_link(request, email):
         resend_email(request, user, email)
         return redirect('/login')
     return redirect('/signup')
+def resend_new_email_activation_link(request, email):
```

Comment:
Also try to test these methods

Reply:
I'm not using a form with this methods so I though testing this **resend_activation_email** would be represent both